lib/ransack/helpers/form_builder.rb

```diff
@@ -6,13 +6,12 @@ module ActionView::Helpers::Tags
   # https://github.com/rails/rails/commit/c1a118a
   class Base
     private
-    if defined? ::ActiveRecord
```

Comment:
I'm not sure that this was ever truly dependent on ActiveRecord.  At one point, this was a version check.  I agree with the comment that it would be nice to avoid this patch to ActionView, but I also think that the patch is necessary regardless of whether ActiveRecord is defined or not.  Moreover, the code within doesn't depend on ActiveRecord as we are just calling `send` on an object 🤔 .